### ISSUE TITLE
[High Contrast] Menu: Flyout command has wrong background color when open

### ISSUE THREAD

**Issue by rigdern** [winjs/winjs#1397]
1. Open `menu.html` from http://1drv.ms/1LlLARL in Edge
2. Switch the machine to use "High Contrast 1"
3. Click the `show menu` button to open the menu
4. Click the `flyout option` button to activate the command and open the Flyout

**Unexpected result:** The command labeled `flyout option` doesn't have a background color. Instead, it should have a green background color (high contrast 1).

![image](https://cloud.githubusercontent.com/assets/199935/9398643/05485b34-475e-11e5-9ad1-474fe4eb1e31.png)
